packages/components/Button/src/__snapshots__/Button.test.tsx.snap

```diff
@@ -5,6 +5,11 @@ exports[`Button default 1`] = `
   acceptsKeyboardFocus={true}
   accessibilityLabel="Default Button"
   accessibilityRole="button"
+  accessibilityState={
+    Object {
+      "disabled": undefined,
```

Comment:
Shouldn't this show up as false if it's not disabled (or not show up at all) ?

Reply:
I feel like state === undefined implies the state isn't supported instead of being a supported state in the "false" setting.  I'd probably `!!` the style prop value

Reply:
or do what you did in RadioButton

Reply:
Good catch. Yeah I agree. I fixed it and updated the snapshot.